RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc

```diff
@@ -10,44 +11,430 @@
 
 #include "RecoTracker/MkFit/interface/MkFitGeometry.h"
 
-#include "createPhase1TrackerGeometry.h"
-
 // mkFit includes
 #include "RecoTracker/MkFitCore/interface/ConfigWrapper.h"
 #include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
 #include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+#include "RecoTracker/MkFitCMS/interface/LayerNumberConverter.h"
 
 #include <atomic>
 
+//------------------------------------------------------------------------------
+
+#include "DataFormats/GeometrySurface/interface/RectangularPlaneBounds.h"
+#include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
+//#include "Geometry/CommonDetUnit/interface/GlobalTrackingGeometry.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetUnit.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/StripGeomDetType.h"
+//#include "Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h"
```

Comment:
any reason to keep these around?

Reply:
I'll remove them, I was basing this on Fireworks Reco geometry extractor and there we also extract "topology".

Reply:
removed